src/components/ChatApp/ChatApp.tsx

```diff
@@ -70,6 +72,18 @@ export default function ChatApp() {
     api.sendMessage(text, image).then((threadId) => {
       if (isFirstMessage) {
         setThreadId(threadId);
+        let allChatsObject = allChats;
+        allChatsObject.allThreadIds = allChatsObject.allThreadIds.concat(
+          threadId
+        );
+        allChatsObject.allChannelIds = allChatsObject.allChannelIds.concat(
+          channelId
+        );
+        allChatsObject.checkbox = allChatsObject.checkbox.concat(false);
+        allChatsObject.text = allChatsObject.text.concat(text);
+        allChatsObject.imageURL = allChatsObject.imageURL.concat(image);
+        setAllChats(allChatsObject);
+        console.log(allChats);
```

Comment:
No need for a console log when merging with master.

Reply:
removed